test/functional/http/post/4.multisignature.advanced.js

```diff
@@ -402,4 +402,17 @@ describe('POST /api/transactions (type 4) register multisignature', () => {
 			pendingMultisignatures
 		);
 	});
+
+	describe('verification', () => {
+		it('sending already confirmed second secret should fail', () => {
```

Comment:
The message is identical to test case above `second secret`. I believe its a typo mistake. It should be multisignature transaction.